client/my-sites/plans/jetpack-plans/i5/products-grid-i5/index.tsx

```diff
@@ -182,6 +182,13 @@ const ProductsGridI5: React.FC< ProductsGridProps > = ( {
 					/>
 				</div>
 			</section>
+
+			<section className="products-grid-i5__section products-grid-i5__asterisk-items">
+				<h2 className="products-grid-i5__asterisk-item">
+					{ translate( 'All plans include priority support' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **2** times:
`translate( 'Personal Plans now Include Priority Support' )` *ES Score: 7*